src/line.rs

```diff
@@ -503,7 +503,7 @@ pub enum Opcode<R: Reader> {
     UnknownStandard1(constants::DwLns, u64),
 
     /// An unknown standard opcode with multiple operands.
-    UnknownStandardN(constants::DwLns, Vec<u64>),
+    UnknownStandardN(constants::DwLns, R),
```

Comment:
To be clear, the issue is the size of `Vec` bloating the decoded instruction size, *not* that we are hitting a lot of unknown standard opcodes that also have more than one operand, right?

Reply:
If so, we could save another word (assuming this is still the widest variant) by boxing `R`, since `R` is usually two words with `EndianBuf`.

Reply:
No, it's not the size of the `Vec`. The largest variant is currently `DefineFile` at 48 bytes, and `UnknownStandardN` was already only 32 before this change.

We're not hitting unknown standard opcodes.

I haven't investigated the assembly in great depth (there's too much), but there's a few inlined `Vec` functions that are no longer needed, and I guess this helps the optimiser. It also means there's no code to run to drop the `Opcode`.